### PR TITLE
Plants config release/103

### PR DESCRIPTION
**Pull request by brunocontrerasmoreira**
## Description

Hi @CristiGuijarro @dthybert @gnaamati  @marcoooo @james-monkeyshines these are the updated config files for Plants Compara for release 103. 

We have updated the species tree, 

- renaming two species which are now strains sets as explained on ticket https://www.ebi.ac.uk/panda/jira/browse/ENSINT-518 (panicum and theobroma)
- adding five new species, including a tetraploid (arachis)

We have also updated the allowed species files, removing some species to make room for the new ones. I am afraid we will do this for the next few releases. 

Let me know if this is ok with Compara,
Bruno

### CODE REVIEW

**Comment by CristiGuijarro** [comment]
`hordeum_vulgare_goldenpromise
ostreococcus_lucimarinus
panicum_hallii_fil2
prunus_avium
theobroma_cacao_criollo`

Have all been removed from the allowed species file, but remain in the species tree -  I am not sure that this won't break something somewhere in either the pipelines or later display?

**Comment by brunocontrerasmoreira** [comment]
I don't know either @CristiGuijarro . This is the first release we edit this file, we'll follow your recommendations

**Comment by CristiGuijarro** [comment]
It might be fine, @JAlvarezJarreta has told me this (and I quote directly):

> for e102 we had species in the species tree that we didn't include in our database and everything was working fine

File: conf/plants/species_tree.topology.nw [hunk 1]
```diff
@@ -177,6 +182,10 @@
             phaseolus_vulgaris,
             glycine_max,
            )Phaseoleae,
+           (
+            arachis_hypogaea_A,
+            arachis_hypogaea_B,
```
Comment (muffato) [inline conf/plants/species_tree.topology.nw:187]: Currently all polyploids are also in the species tree "on their own". Can you add `arachis_hypogaea` too ?

File: conf/plants/allowed_species.json [hunk 2]
```diff
@@ -16,6 +18,7 @@
     "cannabis_sativa_female",
     "capsicum_annuum",
     "chara_braunii",
+    "chenopodium_quinoa"
```
Comment (muffato) [inline conf/plants/allowed_species.json:21]: Missing comma
```suggestion
    "chenopodium_quinoa",
```

**Review by muffato — CHANGES_REQUESTED** [review]
There is a syntax error in the JSON file that needs to be fixed, and I would like `arachis_hypogaea` to be in the species tree on its own

**Review by muffato — APPROVED** [review]

**Review by CristiGuijarro — APPROVED** [review]

**Comment by dthybert** [comment]
arachis_hypogaea is a tetraploid so it means that we will be at 101 haploid genomes and not 100. Could you please remove one diploid species from the set to stay at 100?

**Comment by muffato** [comment]
FYI the current count of haploid genomes is 104, not 100

**Comment by brunocontrerasmoreira** [comment]
Done, I've removed one more

**Comment by muffato** [comment]
Sorry, 102, not 104. (in `ensembl_compara_plants_49_102`)

**Comment by brunocontrerasmoreira** [comment]
Thanks @muffato , does that include outgroups? Can't we leave like that?

**Comment by dthybert** [comment]
> Thanks @muffato , does that include outgroups? Can't we leave like that?

100 is just for plant so you can leave as it is.

**Pull request merged**